WordPress/src/test/java/org/wordpress/android/util/LiveDataUtilsTest.kt

```diff
@@ -66,4 +66,82 @@ class LiveDataUtilsTest : BaseUnitTest() {
         sourceB.value = null
         assertThat(combineMap.value).isEqualTo(mapOf(keyA to valueA))
     }
+
+    @Test
+    fun `when skipping 1 on a LiveData emitting nothing, nothing is emitted`() {
+        // Given
+        val source = MutableLiveData<String>()
+        check(source.value == null)
+
+        val skip = source.skip(1)
+
+        // When
+        var emitCount = 0
+        skip.observeForever {
+            emitCount += 1
+        }
+
+        // Then
+        assertThat(emitCount).isZero()
+        assertThat(skip.value).isNull()
+    }
+
+    @Test
+    fun `when skipping 1 on a LiveData emitting a single value, nothing is emitted`() {
+        // Given
+        val source = MutableLiveData<String>().apply { value = "Alpha" }
+        val skip = source.skip(1)
+
+        // When
+        var emitCount = 0
+        skip.observeForever {
+            emitCount += 1
+        }
+
+        // Then
+        assertThat(emitCount).isZero()
+        assertThat(skip.value).isNull()
+    }
+
+    @Test
+    fun `when skipping 1 on a LiveData emitting multiple values, the first value is not emitted`() {
+        // Given
+        val source = MutableLiveData<String>().apply { value = "Alpha" }
+        val skip = source.skip(1)
+
+        // When
+        var emitCount = 0
+        skip.observeForever {
+            emitCount += 1
+        }
+
+        source.postValue("Bravo")
```

Comment:
For these 2 tests, I wanted to capture the scenario: 

1. `LiveData` **has pre-existing value**
2. An observer is added to `LiveData`
3. `LiveData.postValue()`

Instead of:

1. `LiveData` has no value 
2. An observer is added to `LiveData`
3. `LiveData.postValue()` 
4. `LiveData.postValue()` 

Both scenarios should be covered by `skip()`. I realize this probably really isn't that important but it's nice to have some variance. 

I added a comment in 00ac385f7db54d0262823b4cea5fea3ebe0140c5. Is that alright?